docs/peer_forwarder.md

```diff
@@ -11,18 +11,21 @@ It's configurable for `aggregate` processor using `identification_keys` configur
 Presently peer discovery is provided by either a static list or by a DNS record lookup or AWS Cloudmap.
 
 ### Static discovery mode
+Static discover mode allows Data Prepper node to discover nodes using a list of IP addresses or domain names.
 ```yaml
 peer_forwarder:
   discovery_mode: static
   static_endpoints: ["data-prepper1", "data-prepper2"]
 ```
+
 ### DNS lookup discovery mode
+DNS discovery is recommended when scaling out a Data Prepper cluster. The core concept is to configure a DNS provider to return a list of Data Prepper hosts when given a single domain name.
```

Comment:
> DNS discovery is recommended when scaling out a Data Prepper cluster.

I suggest the following instead:

> We recommend using DNS discovery over static discovery when scaling out a Data Prepper cluster.

We don't recommend DNS discovery exclusively. Cloud Map is a great solution for some AWS customers. But, it is good to recommend the DNS over static list.